Tribler/Test/Core/test_install_dir.py

```diff
@@ -0,0 +1,8 @@
+from Tribler.Core.Utilities.install_dir import determine_install_dir
+from Tribler.Test.Core.base_test import TriblerCoreTest
+
+
+class TriblerCoreTestInstallDir(TriblerCoreTest):
+
+    def test_install_dir(self):
+        self.assertIsInstance(determine_install_dir(), str)
```

Comment:
Maybe we should check that the path returned is the right one? Better to do it now or we will forget about it.